.github/workflows/pr-title-check.yml

```
@@ -0,0 +1,50 @@
+name: PR Title Check
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize, reopened]
+
+permissions:
+  pull-requests: write
+
+jobs:
+  check-pr-title:
+    name: Check PR Title
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check PR Title and Manage Review
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const title = context.payload.pull_request.title;
+            const pattern = /^(feat|fix|build|chore|ci|docs|perf|refactor|revert|style|test):\s\w+:\s[a-z].+/;
+            
+            if (!pattern.test(title)) {
+              await github.rest.pulls.createReview({
+                ...context.repo,
+                pull_number: context.payload.pull_request.number,
+                body: 'Please update the PR title to match the required format: `<type>: <area>: <subject>`\n\nTypes: fix, feat, build, chore, ci, docs, perf, refactor, revert, style, test, chore\nArea: e.g. api, chain, state, mempool, multisig, networking, paych, proving, sealing, wallet, deps\nSubject: start with lowercase',
```

```suggestion
                body: 'Please update the PR title to match the required format: `<type>: <scope>: <subject>`\n\nTypes: build, chore, ci, docs, feat, fix, perf, refactor, revert, style, test\nScope: e.g. api, chain, deps, state, mempool, multisig, networking, paych, proving, sealing, wallet\nSubject: start with lowercase',
```

Alphabetized
Used conventional commit "scope" terminology

Further updates if we do the ``type(scope): summary`` conventional commit syntax.